README.md

```diff
@@ -2,7 +2,15 @@
 
 A development environment for the old PantherHackers Wordpress website.
 
-## Installation
+## Installation - docker
+1.) install [docker](https://docs.docker.com/engine/installation/). Make sure to include `docker-compose` in the installation  
+2.) `cd` into this repo's root directory  
+3.) run `docker-compose up`  
+4.) navigate to [http://localhost:8000](http://localhost:8000) and go through the install  
+5.) If you want to upload data to the database, extract the tables [from mysq to a .sql file](http://stackoverflow.com/a/17479787/2229572) and then come into the root of this repo. 
+use the command `docker exec -it {current_directory}_db_1 mysql -u wordpress -pwordpress wordpress < db.sql` 
```

Comment:
I'm not sure why, but when i run that i get a `ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)` error